### DISCUSSION TITLE
Scammers Previous Usernames

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [ThioJoe/YT-Spammer-Purge#114] [Ideas]
Hi @ThioJoe ,

I don't think this is needed but it's just a idea, 

If people could see the scammers previous youtube usernames, to know which youtuber they were impersonating in the past, and tell that youtuber about them, incase their still impersonating them from their alt.

But, this can be a idea for another project too.

**Comment by ThioJoe** [comment]
Hm yea this could be neat
